### PR TITLE
feat(cli, adapter): update CLI and adapter

### PR DESCRIPTION
**Pull request by Kouprin**
- revisit adapter methods
- add adapter methods for approve and lock eth-to-near txs
- expose token name in adapter methods
- add eth-to-near-find-proof method to CLI
- move transfer commands to testing CLI explicitly

### CODE REVIEW

File: utils/index.js [hunk 1]
```diff
@@ -28,10 +28,18 @@ function getScript (name) {
   return path.resolve(path.join(__dirname, `scripts/${name}.sh`))
 }
 
+function JSONreplacer (key, value) {
+  if (typeof value === 'object' && value !== null && value.type === 'Buffer') {
+    return value.data
+  }
+  return value
+}
```
Comment (frol) [inline utils/index.js:36]: Let's have a better name for it. I was really puzzled reading this function name, unused `key` argument, and the cryptic logic.

File: eth2near/eth2near-block-relay/eth-proof-extractor.js [hunk 2]
```diff
@@ -67,6 +68,65 @@ class EthProofExtractor {
...
+        log = receiptLog
+        break
+      }
+    }
+    if (logFound) {
+      const logEntryData = logFromWeb3(log).serialize()
+      const receiptIndex = proof.txIndex
+      const receiptData = receiptFromWeb3(receipt).serialize()
+      const headerData = proof.header_rlp
+      const _proof = []
```
Comment (frol) [inline eth2near/eth2near-block-relay/eth-proof-extractor.js:108]: Can it have a better name?

**Review by frol — APPROVED** [review]

File: utils/index.js (hunk 1 above)
Comment (Kouprin) [reply utils/index.js:36]: Let me add a comment there.

File: eth2near/eth2near-block-relay/eth-proof-extractor.js (hunk 2 above)
Comment (Kouprin) [reply eth2near/eth2near-block-relay/eth-proof-extractor.js:108]: This is taken from `from-near.js`. We can rename it.

Comment (Kouprin) [reply eth2near/eth2near-block-relay/eth-proof-extractor.js]: Done.

File: utils/index.js (hunk 1 above)
Comment (Kouprin) [reply utils/index.js:37]: Added.

**Pull request merged**